test/helpers/factory.js

```diff
@@ -18,11 +19,11 @@ export function gistData({html, css, javascript, enabledLibraries = []} = {}) {
       content: javascript,
     });
   }
-  if (!isEmpty(enabledLibraries)) {
```

Comment:
I can totally make that change, but it might require something like

```javascript
const popcodeJson = {};
if (!isEmpty(enabledLibraries)) {
  popcodeJSON.enabledLibraries = enabledLibraries;
}
if (!isEmpty(hiddenUIComponents)) {
  popcodeJSON.hiddenUIComponents = hiddenUIComponents;
}
if (!isEmpty(popcodeJSON)) {
  files.push({
    language: 'JSON',
    filename: 'popcode.json',
    content: JSON.stringify(popcodeJSON),
  });
}
```

That way we'd avoid accidentally adding an empty `Array` for `enabledLibraries` or `hiddenUIComponents`. I'm sure there's a nicer way to implement that logic—just sketching it out to express my thoughts.

I modified `gistData` so that it no longer includes default empty `Array` parameters for `enabledLibraries` or `hiddenUIComponents`, so they will be falsey if not passed and `JSON.stringify` won't include them. This would break if anyone ever *purposefully* passed empty `Array`s to `gistData` (because then `popcode.json` would have empty `Array` values), but that's currently not done anywhere `gistData` is used. But it makes sense to me if it should be able to properly handle purposeful empty `Array` arguments. What are your thoughts on that?